lib/routes/ncpssd/newlist.ts

```diff
@@ -0,0 +1,107 @@
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import { URL } from 'url';
+
+export const route: Route = {
+    path: '/newlist',
+    categories: ['government'],
+    example: '/ncpssd/newlist',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: {
+        source: ['ncpssd.cn/', 'ncpssd.cn/newlist?type=0'],
```

Comment:
```suggestion
        source: ['ncpssd.cn/', 'ncpssd.cn/newlist'],
```
search params cannot be matched